src/local-settings.js

```diff
@@ -0,0 +1,38 @@
+import env from './environment'
+import { IPFS_ENDPOINT } from './endpoints'
+
+const DEFAULT_ETH_NODE = 'DEFAULT_ETH_NODE'
```

Comment:
Like in the client, I'd recommend adding a `DASHBOARD_` prefix to these keys and allowing them to be configured via environment variables.

Afterwards, we should also document these like we do in the client: https://github.com/aragon/aragon/blob/master/docs/CONFIGURATION.md

Reply:
we have all the env variables [here](https://github.com/aragon/court-dashboard/blob/a739f7896cb846c0a2e4302fdcb7279c9b28c18e/src/environment.js#L31), do you think that we need to change those names?

Reply:
Nevermind me—this looks all good as-is!